Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReportsAPISentinelConnector/__init__.py

```diff
@@ -158,20 +212,82 @@ def gen_chunks_to_object(data,chunksize=100):
     yield chunk
 
 def gen_chunks(data,log_type):
```

Comment:
Name of the Method and return value are confusing. Name indicates it is for gen_chunks but returning latest event time of events

Reply:
modified the function name